### DISCUSSION TITLE
Whistleblowing - ONE TIME KEY CODE

### DISCUSSION THREAD

**Discussion by cienne1** [globaleaks/globaleaks-whistleblowing-software#3806] [General]
Buongiorno.
Vorrei sapere se è stato implementato il meccanismo di One Time Key Code così come delineato nello schema di Linee guida ANAC, che consente l'inserimento delle segnalazioni orali da parte dell'operatore.
Le linee guida ANAC prevedono infatti che il segnalante che effettua l'accesso alla segnalazione sulla piattaforma inserendo il OTKC, riceve il key code definitivo.

**Comment by evilaliv3** [comment]
Grazie per la domanda.

Stiamo supervisionando il completamento di questa implementazione realizzata da un team indipendente direttamente per conto di Anac.

La funzionalità dovrebbe essere rilasciata ufficialmente nell'arco di due settimane o poco più
